app/models/notifications/also_commented.rb

```diff
@@ -1,17 +1,26 @@
-class Notifications::AlsoCommented < Notification
-  def mail_job
-    Workers::Mail::AlsoCommented
-  end
-  
-  def popup_translation_key
-    'notifications.also_commented'
-  end
+module Notifications
+  class AlsoCommented < Notification
+    def mail_job
+      Workers::Mail::AlsoCommented
+    end
 
-  def deleted_translation_key
-    'notifications.also_commented_deleted'
-  end
+    def popup_translation_key
+      "notifications.also_commented"
+    end
+
+    def deleted_translation_key
+      "notifications.also_commented_deleted"
+    end
+
+    def self.notify(comment, _recipient_user_ids)
+      actor = comment.author
+      commentable = comment.commentable
+      recipients = commentable.participations.map(&:author).select(&:local?) - [commentable.author, actor]
```

Comment:
Same questions as before, do we need to force the collections here or could this be a `joins().where().not()`?